### ISSUE TITLE
Does Vary support bounding box detection?

### ISSUE THREAD

**Issue by NNNNAI** [Ucas-HaoranWei/Vary#74]
Thanks for sharing your wonderful work. I notice that Vary toy supports object detection. Could I give the same object dection promt to Vary and get the detection results?

**Comment by Ucas-HaoranWei** [comment]
No, the vision vocabulary of Vary does not train on the object detection dataset. Maybe you can extract the corresponding weights of Vary-toy for Vary to realize it.
